CHANGELOG.rst

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+0.15.8 (2020-07-19)
+-------------------
+
+- [NEW] Added ``humanize`` week granularity translation for Czech.
+- [FIX] ``arrow.get`` will now parse weekdays with no other inputs, see `#446 <https://github.com/arrow-py/arrow/issues/446>`_.
```

Comment:
I would change this to: "`arrow.get` will now pick sane defaults when weekdays are passed with particular token combinations".